test-data/unit/fine-grained.test

```diff
@@ -9733,3 +9733,16 @@ class C:
 [out]
 ==
 main:5: error: Unsupported left operand type for + ("str")
+[case testNoneAttribute]
+from typing import Generic, TypeVar
```

Comment:
Can you add another test case which is otherwise similar, but prefixed with a `# flags: --strict-optional` comment (you can find examples in other test cases)? Test cases don't enable strict `None` checking by default, for historical reasons, but since it's enabled by default outside tests, and it might affect this test case, having it tested would be useful.

Reply:
Great idea, just pushed a commit to add this test.